talks/_posts/2019-08-30-welcome-to-ritlug.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Fall 2019 - Welcome to RITlug!
+author: Jeffery Russell
+layout: post
+slides: 2019-w01-fall-into-to-linux.pdf
```

Comment:
*intro* to Linux.

Small typo here

Reply:
lol, I can always count on you to fix my spelling :smile: